maquetado/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Wolox</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+	<div class="nav">
+		<div class="nv-logo"><img src="assets/wolox_logo.svg"></div>
```

Comment:
there is no need to wrap the img in a div. Set the style in nv-logo directly to the img tag.
```<img src=".." class="nv-logo" />```